### PR TITLE
Update README.MD

### LINKED ISSUE #1: Update README file
README file should have all needed steps for installing the app. Please fix.

### PR DESCRIPTION
**Pull request by lerabo**

### CODE REVIEW

**Comment by lerabo** [comment]
README file is fixed.

File: README.MD [hunk 1]
```diff
@@ -1,3 +1,24 @@
-## Tutorial on How to Build a Shopping Cart with NodeJS/ Express
+How to Build a Shopping Cart with NodeJS/ Express
+## Getting Started
+To get you started you can simply clone the repository:
 
-This repository holds the source code of my YouTube series on how to create a shopping cart with NodeJS/Express/MongoDB
\ No newline at end of file
+```
+git clone https://github.com/FortechRomania/express-mongo-example-project
```
Comment (VladBo) [inline README.MD]: Change it to your URL

File: README.MD [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+and install the dependencies
+```
+npm install
+```
+
+### Prerequisites
+You need git to clone the repository. You can get git from
+[http://git-scm.com/](http://git-scm.com/).
+
+#### MongoDB
```
Comment (VladBo) [inline README.MD]: No need. Local database is not used here, database from cloud is used.

File: README.MD [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
\ No newline at end of file
+```
+git clone https://github.com/FortechRomania/express-mongo-example-project
+```
+and install the dependencies
+```
+npm install
+```
+
+### Prerequisites
```
Comment (VladBo) [inline README.MD]: Can be removed

**Review by VladBo — CHANGES_REQUESTED** [review]

**Review by lerabo — COMMENTED** [review]
improved

**Review by VladBo — APPROVED** [review]

**Pull request merged**